cms/settings/base.py

```diff
@@ -316,9 +315,9 @@
 
 LANGUAGE_CODE = "en-gb"
 WAGTAIL_CONTENT_LANGUAGES = LANGUAGES = [
-    ("en-gb", _("English")),
-    ("cy", _("Welsh")),
-    ("uk", _("Ukrainian")),
+    ("en-gb", "English"),
+    ("cy", "Welsh"),
+    ("uk", "Ukrainian"),
```

Comment:
best to leave these as translatable. Or, use the spelling in each language. Specifically, `Cymraeg` for Welsh.

Reply:
Addressed in [20c4f5d](https://github.com/ONSdigital/dis-wagtail/pull/136/commits/20c4f5daa1e25748ca92bf084aa758b6c855f761)